### PR TITLE
【OSCP】在 SecretFlow 中实现经典联邦策略 SCAFFOLD

### PR DESCRIPTION
**Pull request by UMR99**
Fixed:  #991 
**Type of change**

- [ ] Add new papers (Please tell us why you think this paper is awesome!)
- [ ] Fix the category of an existing paper/papers (Please tell us the reasons)
- [ ] Add a new tool/primitive/application with a new markdown page (Thank you! Also, please tell us more about this awesome thing!)

**Description**

> Please include a summary of the change and the relevant motivation.

### CODE REVIEW

**Comment by UMR99** [comment]
I have read the CLA Document and I hereby sign the CLA

File: secretflow/ml/nn/fl/backend/torch/strategy/scaffold_1.py [hunk 1]
```diff
@@ -0,0 +1,207 @@
...
+print(secretflow.__file__)
+
+import secretflow as sf
+
+# Check the version of your SecretFlow
+print('The version of SecretFlow: {}'.format(sf.__version__))
+# In case you have a running secretflow runtime already.
+sf.shutdown()
+sf.init(['alice', 'bob', 'charlie'], address='local')
+alice, bob, charlie = sf.PYU('alice'), sf.PYU('bob'), sf.PYU('charlie')
```
Comment (zhangxingmeng) [inline secretflow/ml/nn/fl/backend/torch/strategy/scaffold_1.py:11]: 这里看起来是想写测试代码。我们一般测试是要通过pytest单测来进行测试，可以参考一下其他的单测代码完成一下这部分的单测。这里的代码可以考虑整理成为一个tutorial，后续可以作为文档输出。介绍scaffold的原理以及如何使用

**Comment by zhangxingmeng** [comment]
有空更新一下pr，增加下单测，修复一下ci没通过的问题

**Review by zhangxingmeng — CHANGES_REQUESTED** [review]
Please sync main branch from secretflow, then submit after resolve conflict,thanks

**Pull request closed without merging**